runtime/src/server/middleware/sourcemap_stacktrace.ts

```diff
@@ -0,0 +1,97 @@
+import fs from 'fs';
+import path from 'path';
+import { SourceMapConsumer, RawSourceMap } from 'source-map';
+
+function get_sourcemap_url(contents: string) {
+	const reversed = contents
+		.split('\n')
+		.reverse()
+		.join('\n');
+
+	const match = /\/[/*]#[ \t]+sourceMappingURL=([^\s'"]+?)(?:[ \t]+|$)/gm.exec(reversed);
+	if (match) return match[1];
+
+	return undefined;
+}
+
+const file_cache = new Map<string, string>();
+
+function get_file_contents(path: string) {
+	if (file_cache.has(path)) {
+		return file_cache.get(path);
+	}
+	if (fs.existsSync(path)) {
```

Comment:
you can remove this `if` check. if the file doesn't exist then we'll hit the `catch` block and return `undefined` just the same

Reply:
done